src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -173,13 +173,22 @@ public function isArray(): bool
         return self::VALUE_IS_ARRAY === (self::VALUE_IS_ARRAY & $this->mode);
     }
 
+    /**
+     * Returns true if the option may have either a positive or negative value.
+     *
+     * @return bool true if mode is self::VALUE_NEGATABLE, false otherwise
+     */
     public function isNegatable(): bool
     {
         return self::VALUE_NEGATABLE === (self::VALUE_NEGATABLE & $this->mode);
     }
 
     /**
+     * Sets the default value.
+     *
      * @return void
+     *
+     * @throws LogicException When incorrect default value is given
```

Comment:
We have a lot of logical errors in the codebase, I don't think it's appropriate to add this annotation here. `@throw` is more for runtime exceptions, when the exception may occur at random.

Reply:
Good point. I added it here to match `InputOption`, but I've now gone with removing it on both places.